elks/arch/i86/lib/homebrew.S

```diff
@@ -0,0 +1,17 @@
+// functions for communicating with homebrew computers (INT 0x88 API)
+
+#define ARG0	2
+
+	.code16
```

Comment:
Although likely not currently strictly needed here, lets add `.arch   i8086, nojumps` before `.code16`, to enforce strict early 8088 execution capability.